src/journal.c

```diff
@@ -209,8 +226,16 @@ static void save_to_gnomevfs(Wiggy *wig, const char *filename)
         gtt_ghtml_display(wig->gh, wig->filepath, wig->prj);
         gtt_ghtml_show_links(wig->gh, TRUE);
 
-        gnome_vfs_close(wig->handle);
-        wig->handle = NULL;
+        if (FALSE == g_output_stream_close(G_OUTPUT_STREAM(wig->ostream), NULL, &error))
+        {
+            g_warning("Failed to close HTML file: %s", error->message);
```

Comment:
Perhaps in this case it always is HTML. But the term could be dropped from the message anyway, to keep it more generic.

Reply:
I did so :-)